media/redcore/webservices/joomla/site.users.1.0.0.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * @package     Redcore
+ * @subpackage  Api
+ *
+ * @copyright   Copyright (C) 2005 - 2015 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('JPATH_BASE') or die;
+
+/**
+ * Api Helper class for overriding default methods
+ *
+ * @package     Redcore
+ * @subpackage  Api Helper
+ * @since       1.2
+ */
+class RApiHalHelperSiteUsers
+{
+	/**
+	 * Service for reset password of user
+	 *
+	 * @param   string  $email  Email of user account
+	 *
+	 * @return  boolean         True on success. False otherwise.
+	 */
+	public function forgotPassword($email)
```

Comment:
Thanks for information. I don't know it.

But these function don't need it, just "email" parameter is enough.